<?xml version="1.0"?>
<!--roslaunch pcl_tut filter_downsample.launch-->

<launch>

    <!-- From tutorial -->

    <node name="pcl_read" pkg="pcl_tut" type="pcl_read">
<!--        <param name="path" value="$(find chapter10_tutorials)/data/test_pcd.pcd" />-->
	</node>

	<node name="pcl_filter" pkg="pcl_tut" type="pcl_filter">
	</node>

	<node name="pcl_downsampling" pkg="pcl_tut" type="pcl_downsampling">
	</node>

	<node name="pcl_visualize2" pkg="pcl_tut" type="pcl_visualize2">
<!-- 		<remap from="pcl_partitioned" to="pcl_filtered"/> -->
		<remap from="pcl_partitioned" to="pcl_downsampled"/>
	</node>/>

<!--	<node name="pcl_visualize" pkg="pcl_tut" type="pcl_visualize">-->
<!--		<remap from="pcl_output" to="pcl_downsampled"/>-->
<!--	</node>-->

	<node pkg="tf"
		  type="static_transform_publisher"
		  name="tf_publisher"
		  args="0 0 0 0 0 0 map odom 10" />

		<node name="rviz" pkg="rviz" type="rviz" respawn="true"  />
<!--	output="screen"-->
</launch>
